src/Module/Playback/Stream_Playlist.php

```diff
@@ -297,14 +298,11 @@ private static function media_object_to_url(Media $object, string $additional_pa
                     }
                 }
             } else {
-                if (in_array($type, array('song', 'podcast_episode', 'video'))) {
+                if (in_array($type, [LibraryItemEnum::SONG, LibraryItemEnum::PODCAST_EPISODE, LibraryItemEnum::VIDEO])) {
                     /** @var Song|Podcast_Episode|Video $object */
                     $url['url'] = (!empty($user))
                         ? $object->play_url($additional_params, '', false, $user->id, $user->streamtoken)
                         : $object->play_url($additional_params);
-                } elseif ($type == 'democratic') {
```

Comment:
Democratic doesnt implement the Media interface, so they are filtered out beforehand